12-13-21_temporal_encryption_payload_converter/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Chris Cowell
```

Comment:
Should be copyrighted to Nightfall or Shoreline Labs?

Reply:
Nightfall AI